tests/cairo_files/instructions/test_exchange_operations.cairo

```diff
@@ -0,0 +1,282 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+// Starkware dependencies
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.uint256 import Uint256, assert_uint256_eq
+from starkware.cairo.common.math import assert_nn
+
+// Local dependencies
+from kakarot.instructions.exchange_operations import ExchangeOperations
+from kakarot.model import model
+from kakarot.stack import Stack
+from tests.utils.utils import TestHelpers
+
+// @notice Prepare a stack with `stack_len` elements to test swap logic. 
+func prepare_stack{range_check_ptr}(stack_len : felt, swap_idx: felt, swap_idx_element: Uint256, top_stack_element: Uint256,  stack : model.Stack*) -> (prepared_stack: model.Stack*) {
```

Comment:
can you use the init_stack() wording to be consistent with the init_context() used in others tests? This is not perfect because we abbreviate the initialize word but we should prioritize consistency for now

Reply:
You may have a formatting issue with the stack parameter (2 spaces before colon, 1 after)